samples/batch/transcription-enabled-storage/README.md

```diff
@@ -0,0 +1,49 @@
+# Transcription Enabled Storage
+
+<!-- 
+Guidelines on README format: https://review.docs.microsoft.com/help/onboard/admin/samples/concepts/readme-template?branch=master
+
+Guidance on onboarding samples to docs.microsoft.com/samples: https://review.docs.microsoft.com/help/onboard/admin/samples/process/onboarding?branch=master
+
+Taxonomies for products and languages: https://review.docs.microsoft.com/new-hope/information-architecture/metadata/taxonomies?branch=master
+-->
+
+This project contains a Solution Accelerator which will automatically transcribe audio files added to an Azure Storage Container and writes the results into another container.
+
+See the Contribution guide for how to contribute to this repo.
+
+## Contents
+
+Outline the file contents of the repository. It helps users navigate the codebase, build configuration and any related assets.
```

Comment:
These text look like a template description, but not specific to this project

Reply:
yeah good catch! added this by accident when I updated the readme. Will remove it